pkg/authorization/authz_unix_test.go

```diff
@@ -257,20 +249,14 @@ func (t *authZPluginTestServer) stop() {
 // auth is a used to record/replay the authentication api messages
 func (t *authZPluginTestServer) auth(w http.ResponseWriter, r *http.Request) {
 	t.recordedRequest = Request{}
-	defer r.Body.Close()
-	body, err := ioutil.ReadAll(r.Body)
+	body, _ := ioutil.ReadAll(r.Body)
```

Comment:
the body is checked in the tests above using reflect.DeepEqual so I think it's safe to ignore the error but just print the body to the response (it will fail the reflect check eventually)